logging/log_formatters.go

```diff
@@ -32,55 +32,35 @@ import (
 // ----------
 
 // TextFormatter returns a default logrus.TextFormatter with specific settings
-func TextFormatter(hooks []FormatterHook) logrus.Formatter {
+func TextFormatter() FormatterWrapper {
 	return &AcraTextFormatter{
 		Formatter: &logrus.TextFormatter{
 			FullTimestamp:    true,
 			TimestampFormat:  time.RFC3339,
 			QuoteEmptyFields: true,
 		},
-		Hooks: hooks,
+		Hooks: nil,
```

Comment:
Fields are initialized with `nil` by default so this can be omitted. However, I don't particularly mind explicit initialization either.

Reply:
omit default values is good to keep fewer lines of code, less time to verify formatting, etc ) but anyway didn't find any suggestions/conventions on [effective go](https://golang.org/doc/effective_go.html), so up to you